ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disk-encryption-set/minimal-permission/cucushift-installer-rehearse-azure-ipi-disk-encryption-set-minimal-permission-workflow.yaml

```diff
@@ -0,0 +1,13 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-minimal-permission
+  steps: 
+    pre:
+    - ref: azure-provision-service-principal-minimal-permission
+    - chain: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-provision
+    post:
+    - chain: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-deprovision
```

Comment:
we need `azure-deprovision-sp-and-custom-role` step here?

Reply:
good catch, fixed.